### PR TITLE
Update Enchant.js

### PR DESCRIPTION
**Pull request by Mercoory**
Prevent Enchant bot to be flooded (and beeing muted) with overhead messages

### CODE REVIEW

**Comment by jaenster** [comment]
I tested/used this. It works.

**Comment by noah-** [comment]
can you fix the trailing space and move it inside floodCheck?
it should fail flood check because command[1] doesn't exist which is nick

File: d2bs/kolbot/libs/bots/Enchant.js [hunk 1]
```diff
@@ -493,6 +493,10 @@ MainLoop:
 	this.floodCheck = function (command) {
 		var cmd = command[0],
 			nick = command[1];
+			
+		if (!nick) {	// ignore overhead messages
+				return true;
```
Comment (noah-) [inline d2bs/kolbot/libs/bots/Enchant.js]: please fix the indentation here, other than that LGTM

**Comment by WiZzMaN47** [comment]
I have an enchant.js that works thanks to kolton 👍👀👍

File: d2bs/kolbot/libs/bots/Enchant.js (hunk 1 above)
Comment (Mercoory) [reply d2bs/kolbot/libs/bots/Enchant.js]: Fixed, and thanks for your time :)
Now I have to press "Close pull request" button?

**Pull request merged**